modules/platform/nuxeo-fsexporter/src/main/java/org/nuxeo/io/fsexporter/DefaultExporterPlugin.java

```diff
@@ -138,4 +142,14 @@ protected String getFileName(Blob blob, DocumentModel docfrom, File folder, int
 
         return prefix + blob.getFilename();
     }
+
+    @Override
+    public DocumentModelList getCollectedDocuments(CoreSession session, DocumentModel collection) {
+        Collection collectionAdapter = collection.getAdapter(Collection.class);
+        return collectionAdapter.getCollectedDocumentIds()
+                                .stream()
+                                .map(IdRef::new)
+                                .map(session::getDocument)
+                                .collect(Collectors.collectingAndThen(Collectors.toList(), DocumentModelListImpl::new));
```

Comment:
I'm not a fan of `collectingAndThen` here @kevinleturc. I find it more readable to write

    List<DocumentModel> docs = collection.getAdapter(Collection.class)
                                         .getCollectedDocumentIds()
                                         .stream()
                                         .map(IdRef::new)
                                         .map(session::getDocument)
                                         .collect(Collectors.toList());
    return new DocumentModelListImpl(docs);

But if we inline this in the caller we don't even need the `DocumentModelListImpl` or the `Collectors.toList`.